lib/pbench/test/unit/server/database/test_audit_db.py

```diff
@@ -59,9 +59,9 @@ def test_create(self, fake_db):
     def test_construct_null(self, fake_db):
         """Test handling of Audit record null value error"""
         self.session.raise_on_commit = IntegrityError(
-            statement="", params="", orig=FakeDBOrig("NOT NULL constraint")
+            statement="", params="", orig=FakeDBOrig("not null constraint")
         )
-        with pytest.raises(AuditNullKey, match="'operation': 'CREATE'"):
+        with pytest.raises(AuditNullKey):
```

Comment:
You've removed the matching constraint...should we add an assertion, or is the operation parameter now just gone from the exception?  (You added additional assertions in `test_users_db.py`.)

Ditto for the analogous calls below.